app/templates/profile.html

```diff
@@ -54,7 +60,7 @@
               <div class="modal-dialog profile-modal-dialog profile-modal-fadeIn">
                 <div class="modal-content">
                   <div class="modal-header profile-modal-header">
-                    <h5 class="modal-title">{{ gettext("Update name") }}</h5>
+                    <h1 class="modal-title">{{ gettext("Update name") }}</h1>
```

Comment:
There should be only single `h1` in each page. You can lots of `h2`s under a single `h1`

Reply:
well there is no h1 in the entire profile page but I changed them all to h2 if that's alright.  They are all in the same level